Dockerfile

```diff
@@ -1,13 +1,13 @@
 ARG DOCKER_VERSION=19.03
-ARG PYTHON_VERSION=3.9.0
+ARG PYTHON_VERSION=3.7.9
 
 ARG BUILD_ALPINE_VERSION=3.12
 ARG BUILD_CENTOS_VERSION=7
-ARG BUILD_DEBIAN_VERSION=slim-buster
+ARG BUILD_DEBIAN_VERSION=slim-stretch
```

Comment:
not sure what was the reason previously to have `stretch-20200414-slim`

Reply:
The date just allows more granularity on exactly which image build you want. This is essentially `stretch-latest-slim` as opposed to the version that was build on 2020-04-14